pyproject.toml

```diff
@@ -90,11 +90,11 @@ strict_equality = true
 ignore_missing_imports = true
 
 [tool.cibuildwheel]
-skip = "pp*"
+# Skip musllinux builds for the moment
+skip = "*-musllinux_*"
 test-extras = "test"
 test-command = "pytest {project}/tests"
 manylinux-x86_64-image = "manylinux2014"
```

Comment:
Why are you pinning an old version here? We've moved to manylinux_2_28, and numpy 2.3 only ships 2_28 wheels. Why no numpy 2.3, by the way?

Reply:
There's a bug in awkward array that makes it incompatible with 2.3 for the time being.

Reply:
You can leave 3.9 and 3.10 as manylinux 2014 if you want, that's also an option with overrides.

Reply:
I don't think there's a very good reason to.